db/listen.js

```diff
@@ -0,0 +1,6 @@
+const app = require("./app.js");
+
+app.listen(8080, (err) => {
+  if (err) console.log(err);
+  else console.log("app listening on 8080");
+});
```

Comment:
We don't need listen at this point. So please remove it for now. You can bring it back later when we need it